### ISSUE TITLE
Follow-up with CJIS on any manual pre-publication processes for CIUS

### ISSUE THREAD

**Issue by harrisj** [18F/crime-data-api#528]
It was mentioned briefly in the call with CJIS on 5/30 that they have to do some manual work to present crime data for Puerto Rico since the data provided doesn't align with the database requirements (ORIs and structure). This is why the crime data API has no crime data for Puerto Rico. We can look into any processing we might be able to do with that data in a separate issue, but I wanted to make a note that we should follow up with CJIS to get a rough list of any other pre-production data processing tasks they have before the publication of Crime in the United States, specifically those that amend or alter data within the database, since that would be another way in which our data might differ from what is published.

**Comment by LarryBafundo** [comment]
@harrisj i already followed up with CJIS about this yesterday,  but still waiting on a response.

**Comment by LarryBafundo** [comment]
this issue relates to https://waffle.io/18F/crime-data-explorer/cards/5924c389ccee550020e4d3a7

**Comment by harrisj** [comment]
Larry, you're always one step ahead of me

**Comment by LarryBafundo** [comment]
@harrisj it doesn't seem like we're ever going to get a complete accounting of all of the potential actions that may result in a difference between what the CDE displays and what is published. Do we want to keep at it, or close this issue based on what we know?
